documentation/modules/ROOT/pages/operations/debezium-platform.adoc

```diff
@@ -508,46 +553,63 @@ The pipeline section is the place where you connect the "dots". You can define w
 [.responsive]
 video::cY7TS0nWBoY[youtube, title="Create, edit and remove a pipeline"]
 
-==== Creating pipelines
+==== Creating pipeline
```

Comment:
Use plural nouns with gerund phrases when possible. This should either be _Creating pipelines_, or _Creating a pipeline_.